src/App.js

```diff
@@ -0,0 +1,152 @@
+              </Button>
+            </Navbar.Text>
+          </Navbar.Collapse>
+        </Container>
+      </Navbar>
+
+      <div className="App">
+        <header className="preview">
+          <img src={logo} className="App-logo" id="img-preview" alt="logo" />
+          <p>Drag and Drop or Click to Upload Images</p>
+          <FileUploader
+            name="file"
+            accept={fileTypes}
+            types={fileTypes}
+            multiple={false}
+            classes="Drop-zone"
+            handleChange={(file) => {
+              setImage(file);
+              const reader = new FileReader();
+              reader.onload = (e) => {
+                document.getElementById("img-preview").src = e.target.result;
+              };
+              reader.readAsDataURL(file);
+              document.getElementById("img-preview").style.display = "block";
+              document.getElementById("img-preview").title = file.name;
+            }}
+            label={"Upload or drop a file right here"}
+          />
+          <OverlayTrigger
+            overlay={
+              <Tooltip id="tooltip-disabled">
+                {!image
+                  ? "Select Image to Upload"
+                  : !(token && repository)
+                  ? "Add Token and Repository"
+                  : "Click to Upload"}
+              </Tooltip>
+            }
+            placement="bottom"
+          >
+            <span className="d-inline-block">
+              <Button
+                variant="success"
+                className="mt-3"
+                disabled={!(token && image && repository)}
+              >
+                Upload
+              </Button>
+            </span>
+          </OverlayTrigger>
+        </header>
+        <Modal
+          show={show}
+          onHide={() => {
+            setShow(false);
+          }}
+        >
+          <Modal.Header closeButton>
+            <Modal.Title>GitHub Login</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <Form>
+              <Form.Group className="mb-3">
+                <Form.Label>GitHub Token</Form.Label>
+                <Form.Control type="text" id="token" autoFocus />
+              </Form.Group>
+              <Form.Group className="mb-3">
+                <Form.Label>Repository (username/repository_name)</Form.Label>
+                <Form.Control type="text" id="repo" />
+              </Form.Group>
+            </Form>
+          </Modal.Body>
+          <Modal.Footer>
+            <Button
+              variant="secondary"
+              onClick={() => {
+                setShow(false);
+              }}
+            >
+              Close
+            </Button>
+            <Button
+              variant="success"
+              onClick={() => {
+                let token = document.getElementById("token");
+                let repo = document.getElementById("repo");
+                let validRepo = "[A-za-z0-9_-]+/[A-za-z0-9_-]+";
```

Comment:
I have updated it with;
```js
const validRepo = /^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}\/[a-zA-Z0-9_.-]+$/i;
```
It may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen.